modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -7,29 +7,23 @@
 [id="importing-untrusted-tls-certificates_{context}"]
 = Importing untrusted TLS certificates to {prod-short}
```

Comment:
FYI, I can't find this content in the preview build that the PR prompts me to select for previews. https://62d9769557e45f00589af6f8--eclipse-che-docs-pr.netlify.app/docs/next/hosted-che/hosted-che/

Reply:
Here's the link: https://62d9769557e45f00589af6f8--eclipse-che-docs-pr.netlify.app/docs/next/administration-guide/importing-untrusted-tls-certificates/